# origin: cw (BitKeeper)
# cset: 1.1573 (2.4) key=4230b52ab1OLa1_0ikwHhoMode5Ggg
# URL: http://linux.bkbits.net:8080/linux-2.4/cset@4230b52ab1OLa1_0ikwHhoMode5Ggg
# inclusion: upstream
# descrition: [PATCH] early boot code references check_acpi_pci()
# revision date: Wed, 23 Mar 2005 18:09:48 +0900
#
# S rset: ChangeSet|1.1572..1.1573
# I rset: arch/i386/kernel/earlyquirk.c|1.1..1.2
#
# Key:
# S: Skipped  ChangeSet file only
# O: Original Followed by Updated
# U: Updated  Included with updated range of versions
# I: Included Included verbatim
# E: Excluded Excluded on request from user
# D: Deleted  Manually deleted by subsequent user edit
# R: Revised  Manually revised by subsequent user edit
#
#
# This is a BitKeeper generated diff -Nru style patch.
#
# arch/i386/kernel/earlyquirk.c
#   2005/03/10 18:29:55-03:00 cw@f00f.org +1 -1
#   early boot code references check_acpi_pci()
# 
# ChangeSet
#   2005/03/10 17:59:22-03:00 cw@f00f.org 
#   [PATCH] early boot code references check_acpi_pci()
#   
#   For x86 (and friends) ACPI_BOOT=y (always) and this code wants to call
#   check_acpi_pci().
#   
#   Signed-off-by: Chris Wedgwood <cw@f00f.org>
#   
#   ===== arch/i386/kernel/earlyquirk.c 1.1 vs edited =====
# 
#
===== arch/i386/kernel/earlyquirk.c 1.1 vs 1.2 =====
--- 1.1/arch/i386/kernel/earlyquirk.c	2005-02-18 23:53:58 +09:00
+++ 1.2/arch/i386/kernel/earlyquirk.c	2005-03-11 06:29:55 +09:00
@@ -8,7 +8,7 @@
 #include <asm/pci-direct.h>
 #include <asm/acpi.h>
 
-#ifdef CONFIG_ACPI
+#ifdef CONFIG_ACPI_BOOT
 static int __init check_bridge(int vendor, int device) 
 {
 	/* According to Nvidia all timer overrides are bogus. Just ignore
